test/libsolidity/SolidityNameAndTypeResolution.cpp

```diff
@@ -6140,14 +6140,14 @@ BOOST_AUTO_TEST_CASE(does_not_error_transfer_regular_function)
 {
 	char const* text = R"(
 		contract A {
-			function transfer(uint) {}
+			function transfer() {}
 		}
 
 		contract B {
 			A a;
 
 			function() {
-				a.transfer(100);
+				a.transfer();
```

Comment:
How did this compile before? It errorred but it wasn't even considered a warning?

Reply:
Yes, it errored because the function lookup failed.